cns/client_test.go

```diff
@@ -1251,14 +1260,98 @@ func TestClient(t *testing.T) {
 		}
 		t.Logf("volume:%q deleted sucessfully", volumeID)
 	}
+
+	// Test CnsReconfigVolumePolicy API
+	if spbmPolicyId4Reconfig != "" {
+		var cnsVolumeCreateWithProfileSpecList []cnstypes.CnsVolumeCreateSpec
+		cnsVolumeCreateSpecWithProfile := cnstypes.CnsVolumeCreateSpec{
+			Name:       "pvc-901e87eb-c2bd-11e9-806f-005056a0c9a0-1",
+			VolumeType: string(cnstypes.CnsVolumeTypeBlock),
+			Datastores: dsList,
+			Metadata: cnstypes.CnsVolumeMetadata{
+				ContainerCluster: containerCluster,
+			},
+			BackingObjectDetails: &cnstypes.CnsBlockBackingDetails{
+				CnsBackingObjectDetails: cnstypes.CnsBackingObjectDetails{
+					CapacityInMb: 5120,
+				},
+			},
+		}
+		cnsVolumeCreateWithProfileSpecList = append(cnsVolumeCreateWithProfileSpecList, cnsVolumeCreateSpecWithProfile)
+		t.Logf("Creating volume using the spec: %+v", pretty.Sprint(cnsVolumeCreateWithProfileSpecList))
+		createTask2, err := cnsClient.CreateVolume(ctx, cnsVolumeCreateWithProfileSpecList)
```

Comment:
Can we give meaningful names for the following variable?
`createTask2`, `createTaskInfo2`, `createTaskResult2`, `createVolumeOperationRes2`, `volumeId2`, `volumeCreateResult2`

you can even reuse existing variables.

Reply:
will reuse the variables here